SURVEY.md

```diff
@@ -147,12 +79,94 @@ Add all the questions that pops in your mind we will rank them later on.**
 - Science
 - Other
 
+### 2.6. How big is your company?
+- Just me
+- 2–10
+- 11–50
+- 51–500
+- 501–1,000
+- 1,001–5,000
+- More than 5,000
+- Not sure
+
+### 2.7. How do you work in terms of independence?
+- Work in a team
+- Work on your own project(s)independently
+- Work as an external consultant or trainer
+
+### 2.8. If you work in a team, how big is it?
+- 2–7 people
+- 8–12 people
+- 13–20 people
+- 21–40 people
+- More than 40 people
+
+## 3. Usage of Pandas
+
+### 3.1. On average how often do you use pandas?
+- Everyday
+- Every other day
+- Once a week
+- Once a month
+- Never
+
+### 3.2. Do you use pandas at work?
+- yes
+- no
+
+### 3.3. What do you use Pandas for? (multiple choice/single choice)
+- Data analysis
+- Machine learning
+- Programming of web parsers / scrapers / crawlers
+- Software testing / Writing automated tests
+- Network programming
+- Software prototyping
+- DevOps / System administration / Writing automation scripts
+
+### 3.4. Which of pandas capabilities are most important to you?
+- Loading Data
+- Rename Function
+- Map
+- Apply and Apply Map
+- Shape and Columns
+- Unique function
+- Cross Tab
+- Accessing Sub data frames
+- Plotting
+- Groupby and Statistics
+- Aggregate function
+- Join
+- Masking
+- Handling Missing Values
+
+### 3.5. How would you rate pandas in the following aspects? (Very Satisfied = 5; Satisfied = 4; Neutral = 3; Dissatisfied = 2; Very Dissatisfied = 1)
+- Accuracy
+- Documentation
+- Reliability and stability
+- Usability
+
+## 4. Participation in the pandas mentoring program
 
+### 4.1. On average how often do you do the following? (5 = More than once a day; 4 = Once a day; 3 = Every other day; 2 = Once a week; 1 = Once a month)
+- Read new issues and PRs in the project's repository
+- Comment other people's issues and PRs in the project's repository
+- Submit new PR to the project's repository
 
-### Would you like to be a scholar for pandas-contributors 2019? if yes why??
-### What are your thoughts on pandas-contributors scholarship
+### 4.2. Why did you choose to get involved in the pandas-mentoring program?
+- (open question)
 
+### 4.3. Is this the first open-source project you contribute to?
+- yes
+- no
 
+## 5. Scholarship information
 
+### 5.1. Would you be interested in receiving a scholarship for your contribution to the pandas-mentoring team 2019?
+- yes
+- no
```

Comment:
For me (and the pandas core devs who can find grant opportunities) would be useful to also know information like how many hours per week could be spent in the internship. And it's trickier, but information on how much money we need for the scholarships. Programs like GSoC and others decide it for us, but for NumFOCUS grants, grants directly from pandas funds, or corporate partnerships, would be useful to know.

Reply:
great to know! I can add them to this PR